Controller/ProfilerController.php

```diff
@@ -128,17 +125,6 @@ private function explainSQLServerPlatform(Connection $connection, array $query):
 
         $stmt = $connection->executeQuery($sql, $params, $query['types']);
 
-        // DBAL 2.13 "forward compatibility" BC break handling
-        if ($stmt instanceof Result) {
```

Comment:
so I guess with DBAL 3 this never worked :thinking:

Reply:
Then let's just throw exception in this function to make it clear this is not supported for SQLSRV? Because I'm pretty sure it ccannot work without nextRowset()

Reply:
removed + added an exception